quiche/quic/core/quic_types.h

```diff
@@ -794,7 +794,7 @@ struct QUICHE_EXPORT QuicOwnedPacketBuffer : public QuicPacketBuffer {
 
   QuicOwnedPacketBuffer(QuicOwnedPacketBuffer&& owned_buffer)
       : QuicPacketBuffer(std::move(owned_buffer)) {
-    // |owned_buffer| does not own a buffer any more.
+    // |owned_buffer| does not own a buffer anymore.
```

Comment:
I used a heuristic which I think was right (e.g. here) more often than it was wrong (there's one place below where I got it wrong, which I'll fix after I finish leaving these notes -- sorry)
https://www.grammarly.com/blog/anymore-vs-any-more/